lonestar/analytics/distributed/README.md

```diff
@@ -140,8 +145,9 @@ Examples for Running Provided Apps
 
 To run 3 processes all on a single machine, use the following:
 `GALOIS_DO_NOT_BIND_THREADS=1 mpirun -n=3 ./bfs_push rmat15.gr -graphTranspose=rmat15.tgr -t=4 -num_nodes=1 -partition=oec`
-Note that `-num_nodes=1` is not correct if `ENABLE_GPU` is not set (it does not
-appear as an option in that case).
+Note that `-num_nodes=1` is not correct if heterogeneous execution is not
```

Comment:
The wording here seems a little confusing to me (though I'm not super familiar with the interface). Would something like: "Note: when heterogeneous execution is enabled, the setting `-num_nodes=1` is invalid and will also not appear as an option." mean the same thing here and be a little clearer? 

Optionally afterwards, "Heterogeneous execution is enabled by setting the `GALOIS_CUDA_CAPABILITY` build option."

Reply:
Note: when heterogeneous execution is **not** enabled, `-num_nodes=1` is invalid and will not appear as an option.

Yes, much clearer. My bad for the poor wording.